lib/image_optim/bin_resolver/bin.rb

```diff
@@ -98,6 +98,8 @@ def version_string
         when :jpegrescan
           # jpegrescan has no version so use first 8 characters of sha1 hex
           Digest::SHA1.file(path).hexdigest[0, 8] if path
+        when :guetzli
+          capture("#{escaped_path} -version 2> #{Path::NULL}")[/.*/]
```

Comment:
There is unfortunately no any version flag, while this will currently work as all output goes to err, it is better to use `/\d+(\.\d+)+/`